manifests/i/IDRIX/VeraCrypt/1.25.4/IDRIX.VeraCrypt.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: IDRIX.VeraCrypt
+PackageVersion: 1.25.4
+PackageLocale: en-US
+Publisher: IDRIX
+PackageName: VeraCrypt
+License: Apache 2.0

```

Comment:
```suggestion
License: Apache 2.0
LicenseUrl: https://veracrypt.fr/en/VeraCrypt%20License.html
Copyright: Copyright © 2013-2019 IDRIX. All rights reserved.
CopyrightUrl: https://veracrypt.fr/en/Legal%20Information.htm
```

Reply:
Are you sure about the `All rights reserved`? It is Apache 2.0 after all.

Reply:
> Are you sure about the `All rights reserved`? It is Apache 2.0 after all.

Yes, If you go to the copyright URL it clearly says all rights reserved. It just means that they reserve the rights that they haven't given away under the license